schema/failure.yaml

```diff
@@ -21,34 +26,16 @@ properties:
       - authentication
       - authorization
       - connectivity
-      - delegate-provioning
+      - delegate-provisioning
       - delegate-restart
       - input-timeout
       - policy-evaluation
       - timeout
       - unknown
       - verification
-  exit_codes:
-    description: Exit Codes specifies the exit codes.
-    type: array
-    items:
-      type: string
+      - user-mark-fail
 
 oneOf:
-- allOf:
-  - properties:
-      type:
-        const: abort
-  - properties:
-      spec:
-        $ref: ./failure_abort.yaml
-- allOf:
-  - properties:
-      type:
-        const: ignore
-  - properties:
-      spec:
-        $ref: ./failure_ignore.yaml
```

Comment:
can you provide some backgaround as to why these are being removed?  we need a way to fast-fail and early exits. will these be replaced with some other syntax to enable those capabilities?

Reply:
@bradrydzewski we are not removing this strategies just that it doesn't any spec, thus only type field is enough. This is what we have in V0 as well, let me know if you think we should have empty spec as well